File: modul-4/Soal-4/PRAK404-2310817110013-Randy_Febrian.py
while(1):
    print("pilih program")
    print("1. Penjumlahan")
    print("2. Pengurangan")
    print("3. Perkalian")
    print("4. Pembagian")
    print("5. Exit")
    operasi = int(input("Masukkan pilihan : "))
    if(operasi == 5):
        print("Terimakasih, telah menggunakan kalkulator Randy Febrian")
        break
    elif(operasi > 5 or operasi < 1):
        print("Input anda salah, silahkan coba lagi\n")
        continue

    nilai1 = float(input("Masukkan nilai pertama : "))
    nilai2 = float(input("Masukkan nilai kedua : "))

    if operasi == 1:
        hasil = nilai1 + nilai2
        print("hasil Penjumlahan antara %.2f dengan %.2f adalah %.2f\n" % (nilai1, nilai2, hasil))
    elif operasi == 2:
        hasil = nilai1 - nilai2
        print("hasil Pengurangan antara %.2f dengan %.2f adalah %.2f\n" % (nilai1, nilai2, hasil))
    elif operasi == 3:
        hasil = nilai1 * nilai2
        print("hasil Perkalian antara %.2f dengan %.2f adalah %.2f\n" % (nilai1, nilai2, hasil))
    elif operasi == 4:
        hasil = nilai1 / nilai2
        print("hasil Pembagian antara %.2f dengan %.2f adalah %.2f\n" % (nilai1, nilai2, hasil))

File: modul-4/Soal-4/PRAK404-2310817110013-Randy_Febrian.c
#include <stdio.h>

int main(){
    float nilai1, nilai2, hasil;
    int operasi;

    while(1){
        printf("pilih program\n");
        printf("1. Penjumlahan\n");
        printf("2. Pengurangan\n");
        printf("3. Perkalian\n");
        printf("4. Pembagian\n");
        printf("5. Exit\n");

        printf("Masukkan Pilihan : ");
        scanf("%d", &operasi);
            if(operasi == 5){
            printf("Terimakasih, telah menggunakan kalkulator Randy Febrian \n");
            break;
        }
            else if(operasi > 5 || operasi < 1){
                printf("Input anda salah, silahkan coba lagi\n\n");
                continue;
            }        
        printf("Masukkan nilai pertama : ");
        scanf("%f", &nilai1);
        printf("Masukkan nilai kedua : ");
        scanf("%f", &nilai2);

        if(operasi == 1){
            hasil = nilai1 + nilai2;
            printf("Hasil Penjumlahan antara %.2f dengan %.2f adalah %.2f\n\n", nilai1, nilai2, hasil);
        }
        else if(operasi == 2){
            hasil = nilai1 - nilai2;
            printf("Hasil Pengurangan antara %.2f dengan %.2f adalah %.2f\n\n", nilai1, nilai2, hasil);
        }
        else if(operasi == 3){
            hasil = nilai1 * nilai2;
            printf("Hasil Perkalian antara %.2f dengan %.2f adalah %.2f\n\n", nilai1, nilai2, hasil);            
        }
        else if(operasi == 4){
            hasil = nilai1 / nilai2;
            printf("Hasil Pembagian antara %.2f dengan %.2f adalah %.2f\n\n", nilai1, nilai2, hasil);                
        }
    }
    return 0;
}